lib/windows/app/actions/deviceActions.js

```diff
@@ -165,7 +165,8 @@ function logDeviceListerError(error) {
         // to a libusb-compatible-driver will fail to enumerate and trigger a
         // LIBUSB_* error. This is the case of a nRF52840 in DFU mode.
         // We don't want to show an error to the user in this particular case.
-        if (error.message === 'LIBUSB_ERROR_NOT_FOUND' && error.usb.deviceDescriptor) {
+        // errorCode 105 = LIBUSB_ERROR_NOT_FOUND
+        if (error.errorCode === 105 && error.usb.deviceDescriptor) {
```

Comment:
Instead of adding comments describing what the error codes mean, an alternative solution could be to export the errors as constants from the nrf-device-lister-js library. The node-usb library already does this, so it is possible to do f.ex:

```
const usb = require('usb');

...

if (error.errno === usb.LIBUSB_ERROR_NOT_FOUND) { ... }
```

If nrf-device-lister-js did something similar, the error handling would perhaps be more readable.